users/api/http/responses.go

```diff
@@ -30,3 +33,20 @@ func (res tokenRes) Headers() map[string]string {
 func (res tokenRes) Empty() bool {
 	return res.Token == ""
 }
+
+type identityRes struct {
+	Email    string                 `json:"email"`
+	Metadata map[string]interface{} `json:"metadata,omitempty"`
+}
+
+func (res identityRes) Code() int {
+	return http.StatusCreated
+}
+
+func (res identityRes) Headers() map[string]string {
+	return map[string]string{}
+}
+
+func (res identityRes) Empty() bool {
+	return res.Email == "" && len(res.Metadata) == 0
```

Comment:
Just return `false` here. If there is anything missing (if an error has happened) `encodeError` will be called, where this `Empty` method doesn't matter. We use empty only to indicate if there is any content or not (for example when you create user, there is nothing to return on success, so Empty method of that response returns `true`).